config/site.php

```diff
@@ -69,6 +54,26 @@
                         'options' => ['target' => '_blank'],
                         'title' => __('CakeFest'),
                     ],
+                    'newsletter' => [
+                        'url' => [
```

Comment:
This should be a string URL https://cakephp.org/pages/newsletter, since it's from another domain.

Reply:
Done